experiments/lines_of_code/hl.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
the code in this directory was just used to count lines of code, to gather productivity numbers